indico/web/client/js/react/components/pictures/PictureArea.jsx

```diff
@@ -0,0 +1,238 @@
+                                  icon={captureButtonIcon}
+                                  content={captureButtonText}
+                                  onClick={() => onOpenCameraDialog()}
+                                  disabled={!isCameraAllowed}
+                                />
+                              </span>
+                            }
+                            content={
+                              isCameraAllowed
+                                ? Translate.string('Take a picture with your camera')
+                                : Translate.string(
+                                    'You need to grant this site camera permissions before taking a picture'
+                                  )
+                            }
+                          />
+                        </Grid.Column>
+                      </Grid>
+                    )}
+                  </>
+                )}
+                <Divider horizontal />
+                <p>{descriptionText}</p>
+              </Grid.Column>
+            </Grid.Row>
+          </Grid>
+        )}
+        {(fileRejections.length > 0 || customFileRejections) && (
+          <Message negative>
+            <Message.Header>
+              <Translate>There were some problems with the picture</Translate>
+            </Message.Header>
+            {fileRejections.map(({file, errors}) => (
+              <Message.List key={file.path}>
+                {errors.map(e => (
+                  <Message.Item key={e.code}>
+                    {file.name}:- {e.message}
+                  </Message.Item>
+                ))}
+              </Message.List>
+            ))}
+            {customFileRejections && (
+              <Message.List>
+                <Message.Item>{customFileRejections.message}</Message.Item>
+              </Message.List>
+            )}
+          </Message>
+        )}
+      </Segment>
+    </div>
+  );
+}
+
+const pictureDetailsShape = PropTypes.shape({
+  filename: PropTypes.string,
+  size: PropTypes.number,
+  uuid: PropTypes.string,
+  imageSrc: PropTypes.string,
+  upload: PropTypes.shape({
+    failed: PropTypes.bool.isRequired,
+    ongoing: PropTypes.bool.isRequired,
+    finished: PropTypes.bool.isRequired,
+    progress: PropTypes.number.isRequired,
+  }),
+});
+
+PictureArea.propTypes = {
+  dropzone: dropzoneShape.isRequired,
+  capture: PropTypes.object.isRequired,
+  cropper: PropTypes.object.isRequired,
+  picture: pictureDetailsShape,
+  dragText: PropTypes.string,
+  descriptionText: PropTypes.string,
+  picturePlaceholder: PropTypes.string,
+  pictureAction: fileActionShape,
+  picturePreview: PropTypes.func.isRequired,
+  customFileRejections: PropTypes.object,
+};
+
+PictureArea.defaultProps = {
+  picture: null,
+  dragText: Translate.string('Drag a picture here'),
+  descriptionText: Translate.string('Allowed file types: .jpg, .png, .gif.'),
```

Comment:
This default is not used anywhere, so having a translatable string here wastes translators' time. Maybe default to empty instead?

Reply:
Okay, defaulting to empty for the `descriptionText` but leaving the `dragText` since that is being used